addons/website/static/src/js/menu/seo.js

```diff
@@ -230,7 +203,10 @@ var Preview = Widget.extend({
     init: function (parent, options) {
         this.title = options.title;
         this.url = options.url;
-        this.description = options.description || "[ The description will be generated by google unless you specify one ]";
+        this.description = options.description || _t("The description will be generated by search engines based on page content unless you specify one.");
+        if (this.description.length > 160) {
+            this.description = this.description.substring(0,159) + '…';
```

Comment:
Can you do the same for the Social Network preview as you added the description in it?

Reply:
Good idea